include/muu/integer.h

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+//
+
+#pragma once
+
+//! \file
+//! \brief Shorthands for fixed width integer types.
+
+#include "impl/std_type_traits.h"
+
+namespace muu
+{
+	using i8  = char;
+	using i16 = short;
+	using i32 = int;
+	using i64 = long long;
+	using u8  = unsigned char;
+	using u16 = unsigned short;
+	using u32 = unsigned int;
+	using u64 = unsigned long long;
+
+	static_assert (std::is_arithmetic_v<i8>  && std::is_signed_v<i8>    && sizeof(i8)  == 1);
+	static_assert (std::is_arithmetic_v<i16> && std::is_signed_v<i16>   && sizeof(i16) == 2);
+	static_assert (std::is_arithmetic_v<i32> && std::is_signed_v<i32>   && sizeof(i32) == 4);
+	static_assert (std::is_arithmetic_v<i64> && std::is_signed_v<i64>   && sizeof(i64) == 8);
+	static_assert (std::is_arithmetic_v<u8>  && std::is_unsigned_v<u8>  && sizeof(u8)  == 1);
+	static_assert (std::is_arithmetic_v<u16> && std::is_unsigned_v<u16> && sizeof(u16) == 2);
+	static_assert (std::is_arithmetic_v<u32> && std::is_unsigned_v<u32> && sizeof(u32) == 4);
+	static_assert (std::is_arithmetic_v<u64> && std::is_unsigned_v<u64> && sizeof(u64) == 8);
+}
```

Comment:
```suggestion
namespace muu
{
	inline namespace integer_aliases
	{
		using i8  = int8_t;
		using i16 = int16_t
		using i32 = int32_t;
		using i64 = int64_t;
		using u8  = uint8_t;
		using u16 = uint16_t;
		using u32 = uint32_t;
		using u64 = uint64_t;
	}
}
```
- Aliasing the `<cstdint>` typedefs is a more portable and idiomatic means of doing this.
- Those aliases are [already in the muu namespace](https://github.com/marzer/muu/blob/cfe50701eb8bc45193c8c61043a73986e3f8408f/include/muu/fwd.h#L211-L241) so we can bring them in without the noisy `std::` qualifiers
- Sticking them in an inline namespace means other codebases can do `using namespace muu::integer_aliases;` as a nice convenience